### ISSUE TITLE
Filter By User

### ISSUE THREAD

**Issue by corrado-i** [netfirms/appnote#1]
Hello, 
this sample fetch notes of all users. 
Do you know how to filter notes by user?

**Comment by netfirms** [comment]
@corrado16 yes you can, AppWrite can filter data with specific parameter [getDocument](https://appwrite.io/docs/client/database?sdk=web#getDocument)

**Comment by corrado-i** [comment]
But this is by documentId. If I pass the userId works?

**Comment by netfirms** [comment]
It can filter by any param which you declared in model class.

**Comment by corrado-i** [comment]
No sorry. I didn' t explain myself well. I mean by user permission. When you create a document you have the possibility to set the user permission indicating for read and write the user for eg. user:56386eyjshsh . 
So I want filter by this. Can you help me? Sorry for the disturb...

**Comment by netfirms** [comment]
@corrado16  I'm not sure about it, but usually, Appwrite only allows you to get only your document in which you have the permission.

**Comment by corrado-i** [comment]
Ah okay. Thanks for your answer.
Have a good day 😊
